content/docs/getting_started/visual_story/animating_elements.md

```diff
@@ -189,7 +189,7 @@ We'll start by animating the entrance of the banner text and have it "whoosh in"
 Pets can lower your stress levels!</p>
 ```
 
-Reload your story page in your browser, and verify that the banner whooshes in. 
+Reload your story page in your browser, and verify that the banner whooshes in.
```

Comment:
This seems to be something your editor does :) Generally a good thing, but makes it very hard to read which change in this PR was intentional. OK for now, but would be good to prevent in the next ones.